pkg/plugins/policies/meshretry/api/v1alpha1/validator_test.go

```diff
@@ -169,6 +169,19 @@ to:
         retryOn: 
           - 500
           - 409
+`),
+			Entry("target MeshHTTPRoute", `
```

Comment:
What about
```yaml
targetRef:
  kind: MeshHTTPRoute
  name: route-1
to:
  - targetRef:
      kind: MeshService
      name: backend
    default:
      http:
        retryOn: 
          - 500
          - 409
```
I don't think we should allow anything other than `Mesh` in `to` when top level kind is `MeshHTTPRoute`